src/rgw/rgw_process.cc

```diff
@@ -218,7 +218,9 @@ int process_request(RGWRados* const store,
 
   int http_ret = s->err.http_ret;
   int op_ret = 0;
+  string op_name = "unknown";
```

Comment:
Well, I guess one can setup some conditionals directly in the dout, but I'm not sure whether the added complexity will make up for added efficiency. The difficulty comes from the fact that this part of the code can also be reached via some error exits when things have gone wrong before, so I wanted to rather be on the safe side. Not sure whether the tests will handle all error situations so that one could be sure that there are no possibilities for core dumps introduced.

Feel free to propose an update, though.